.github/workflows/build-npm.yml

```diff
@@ -1,39 +1,41 @@
 # This workflow will do a clean install of node dependencies, build the source code,
-# run unit tests, and perform a SonarCloud scan.
+# run unit tests, perform a Sonarqube scan, and publish NPM artifacts from master/main.
 
 # For more information see:
 # https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
-# Common FOLIO configurable env:
+# Common FOLIO configurable environment variables to set:
 #   - YARN_TEST_OPTIONS (options to pass to 'yarn test')
-#   - SQ_ROOT_DIR (where to scan relative to top-level directory)
-#   - FOLIO_NAME
+#   - SQ_ROOT_DIR (root SQ directory to scan relative to top-level directory)
 #   - PUBLISH_MOD_DESCRIPTOR (boolean 'true' or 'false')
 #   - COMPILE_TRANSLATION_FILES (boolean 'true' or 'false')
 
 
 
 name: buildNPM Snapshot
 on: [push, pull_request]
+
 jobs:
-  build-npm:
+  github-actions-ci:
     env:
-      YARN_TEST_OPTIONS: '--ci --coverage --colors'
+      YARN_TEST_OPTIONS: ''
       SQ_ROOT_DIR: './src'
-      PUBLISH_MOD_DESCRIPTOR: 'false'
-      COMPILE_TRANSLATION_FILES: 'false'
+      COMPILE_TRANSLATION_FILES: 'true'
+      PUBLISH_MOD_DESCRIPTOR: 'true'
       FOLIO_NPM_REGISTRY: 'https://repository.folio.org/repository/npm-folioci/'
       FOLIO_NPM_REGISTRY_AUTH: '//repository.folio.org/repository/npm-folioci/'
       FOLIO_MD_REGISTRY: 'https://folio-registry.dev.folio.org'
       NODEJS_VERSION: '18'
       JEST_JUNIT_OUTPUT_DIR: 'artifacts/jest-junit'
       JEST_COVERAGE_REPORT_DIR: 'artifacts/coverage-jest/lcov-report/'
+      BIGTEST_JUNIT_OUTPUT_DIR: 'artifacts/runTest'
+      BIGTEST_COVERAGE_REPORT_DIR: 'artifacts/coverage/lcov-report/'
```

Comment:
Are these still necessary for the pipeline if a repo doesn't use Bigtest? Can they be removed?

Reply:
Obviously, this all shortcoming in the template repo that these updates needed to be made - If this happens, please bring this up on the stripes channel or somewhere public so that supporting teams can get eyes on it and address the issue.